src/view/use-sensor-marshal/use-sensor-marshal.js

```diff
@@ -171,7 +171,7 @@ function tryStart({
     return null;
   }
 
-  const entry: DraggableEntry = registry.draggable.getById(draggableId);
+  const entry: DraggableEntry = registry.draggable.getById(draggableId, sourceEvent.srcElement.shadowRoot || document);
   const el: ?HTMLElement = findDraggable(contextId, entry.descriptor.id);
```

Comment:
You added a param "documentFragment" to findDraggable but the caller does not pass in the argument. Change it to findDraggable(contextId, entry.descriptor.id, sourceEvent.srcElement.shadowRoot || document) works for me

Reply:
Thanks @kehengz, uploaded an additional commit which also fixes this part. The first commit was more of a hacky sketch.
Out of curiosity, did you tested with React 17 or lower?

Reply:
Got it thanks! My repo uses React 17